helper/backup_helper.go

```diff
@@ -34,6 +35,7 @@ func doBackupAgent() error {
 
 	preloadCreatedPipes(oidList, *copyQueue)
 	var currentPipe string
+	var errBuf bytes.Buffer
```

Comment:
1. Is the backup case identical to restore case in terms of errBuf handling? 
2. Do we make separate local buffer for each BackupPipeWriter because here is no any readers[contentToRestore] analog?

Reply:
> What are the reasons of using local version? Refactoring?

Previously the `errBuf` was global as it was used in both restore and backup parts. Now `restore` part doesn't use it, and there is no reason to keep this var global anymore. I consider having a global variable, when it is not required by the logic, is a defect. Thus, I changed this part as well.

Of course we can revert the part related to the backup helper. That will make the delta smaller.

Do you think it is worth to revert the part related to the backup helper?

Reply:
Can backup helper potentially face the similar issues as in restore case?

Reply:
> Can backup helper potentially face the similar issues to restore's problems?

Right now it looks that backup helper should not face such problems, because in `doBackupAgent` function `getBackupPipeWriter` is called only once. So, there can be only one plugin process per backup helper, and one `errBuf` should be enough .

Reply:
Ok, then. Just add more rationale for backup side changes in the PR.

Reply:
Ok, updated the description.